converter/__init__.py

```diff
@@ -47,4 +48,6 @@ def create_app(test_config=None):
     
     bootstrap = Bootstrap(app)
 
+    os.chmod('converter/xml2abc.py', stat.S_IRWXU)
```

Comment:
Wait a second, I don't think you should do this chmd on the fly. Why can't you change the permission using git?

The point I am trying to make is that you are changing the mode in runtime, but this file is static - this doesn't strike me as the correct approach.